README.md

```diff
@@ -47,31 +47,7 @@ dotnet add package SwedbankPay.Sdk
 ```
 
 Configure the SDK in one line.
-
-```csharp
-public void ConfigureServices(IServiceCollection services)
-{
-    services.AddSwedbankPayClient(ApiUrl, AuthenticationToken);
-    ...
-}
-```
-
-This will add the `ISwedbankPayClient` to the system, as well as interfaces for
-all api's in case you want to have more control over what is injected.
-See the [samples][samples] for inspiration and usage.
-Using this to configure the `SwedbankPay.Sdk` will set up a
-`LoggingDelegatingHandler` that aids in logging error responses from the api.
-
-
-## Getting started
-
-Install the `SwedbankPay.Sdk` NuGet in your project:
-
-```
-dotnet add package SwedbankPay.Sdk
-```
-
-Configure the SDK in one line.
+(This requires that you have added `SwedbankPay.Sdk.Extensions`)
```

Comment:
```suggestion
```

Reply:
Without the Extensions NuGet we plan to create there is no way to configure it in one line.